### PR TITLE
Vote-768: Quicklinks Style

### PR DESCRIPTION
**Pull request by clmedders**
## Jira ticket

[Vote-768](https://cm-jira.usa.gov/browse/VOTE-768)

## Description

Adding styles for quick links section and twig template for section as well

### Pre-deploy

1. `lando retune`

### Post-deploy

n/a

### QA/Test

1. Pull down branch and start local be sure to run `lando retune` to capture config 
2. Visit home page and verify that quick links section to match [figma design](https://www.figma.com/file/E70piAGXmi4UzrHs0A90cm/Vote.gov-Redesign-2024-(Master)?node-id=3236:15993&mode=dev) has been styled and is response with both mobile and desktop 

## Checklist for the Developer

- [x] A link to the JIRA ticket has been included above.
- [x] No merge conflicts exist with the target branch.
- [x] Automated tests have passed on this PR.
- [x] A reviewer has been designated.
- [x] Deployment and testing steps have been documented above, if applicable.

## Checklist for the Peer Reviewers

- [x] The file changes are relevant to the task objective.
- [x] Code is readable and includes appropriate commenting.
- [x] Code standards and best practices are followed.
- [x] QA/Test steps were successfully completed, if applicable.
- [x] Applicable logs are free of errors.

### CODE REVIEW

File: web/themes/custom/votegov/src/sass/components/quick-links.scss [hunk 1]
```diff
@@ -0,0 +1,63 @@
+@use "uswds-core" as *;
+@use "variables" as *;
+
+.vote-quicklinks {
+  background-color: #11385B;
```
Comment (clmedders) [inline web/themes/custom/votegov/src/sass/components/quick-links.scss:5]: Let me know if i need to add this to the variables file like the hero section... i found the uswds mixin for color but its a little bit different so wasnt sure if i need to do that or stick with the one listed in the design.

File: web/themes/custom/votegov/src/sass/components/quick-links.scss [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    @include grid-col(6);
+    @include u-order(3);
+}
+
+    p {
+      padding-bottom: 20px;
+    }
+
+    a {
+      color: #0AAAE2
```
Comment (rayestrada) [inline web/themes/custom/votegov/src/sass/components/quick-links.scss:61]: There is a `$ac-light-cool` color that you can use after you uncomment it in the variables file.

File: web/themes/custom/votegov/src/sass/components/quick-links.scss [hunk 3]
```diff
@@ -0,0 +1,63 @@
+@use "uswds-core" as *;
+@use "variables" as *;
+
+.vote-quicklinks {
+  background-color: #11385B;
+  color: #FFF;
```
Comment (rayestrada) [inline web/themes/custom/votegov/src/sass/components/quick-links.scss:6]: replace with `$base-white`

File: web/themes/custom/votegov/src/sass/components/quick-links.scss [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  
+  @include at-media('tablet-lg') {
+    @include grid-col(4);
+    @include u-order(1);
+    @include u-padding-y(2);
+    padding-left: 30px;
+  }
+
+    h2 {
+      color: #FFF;
```
Comment (rayestrada) [inline web/themes/custom/votegov/src/sass/components/quick-links.scss:32]: replace with `$base-white`

**Review by rayestrada — CHANGES_REQUESTED** [review]

File: web/themes/custom/votegov/src/sass/components/quick-links.scss (hunk 1 above)
Comment (clmedders) [reply web/themes/custom/votegov/src/sass/components/quick-links.scss:5]: Updated to $base-dark

File: web/themes/custom/votegov/src/sass/components/quick-links.scss (hunk 3 above)
Comment (clmedders) [reply web/themes/custom/votegov/src/sass/components/quick-links.scss:6]: Udpated to $base-white

File: web/themes/custom/votegov/src/sass/components/quick-links.scss (hunk 4 above)
Comment (clmedders) [reply web/themes/custom/votegov/src/sass/components/quick-links.scss:32]: Replaced with $base-white

File: web/themes/custom/votegov/src/sass/components/quick-links.scss [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+    }
+}
+
+.vote-quicklinks__red-line {
+    border-bottom: 4px solid #C74345;
```
[the remark this answers is not in the document]
Comment (clmedders) [reply web/themes/custom/votegov/src/sass/components/quick-links.scss:37]: Added the $ac-warm-active variable

File: web/themes/custom/votegov/src/sass/components/quick-links.scss (hunk 2 above)
Comment (clmedders) [reply web/themes/custom/votegov/src/sass/components/quick-links.scss:61]: Add the $ac-light-cool to links and uncommented from variables file

File: web/themes/custom/votegov/templates/paragraph/paragraph--quicklinks.html.twig [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+<section class="vote-quicklinks">
+  <div class="vote-quicklinks__container">
+    <div class="vote-quicklinks__heading">
+      <h2>
+        {{ content.field_heading | field_value }}
+      </h2>
+    </div>
+    {# <div class="vote-quicklinks__red-line">test</div> #}
+    <hr class="vote-quicklinks__red-line">
```
[the remark this answers is not in the document]
Comment (clmedders) [reply web/themes/custom/votegov/templates/paragraph/paragraph--quicklinks.html.twig:16]: removed line and added with css

**Review by rayestrada — APPROVED** [review]

**Pull request merged**